docs/frameworks/laravel.md

```diff
@@ -58,9 +59,9 @@ We will also need to customize the location for compiled views, as well as custo
 ```dotenv
 VIEW_COMPILED_PATH=/tmp/storage/framework/views
 
-# We cannot store sessions to disk: if you don't need sessions (e.g. API)
-# then use `array`, else store sessions in database or cookies
-SESSION_DRIVER=array
+# We cannot store sessions to disk: if you don't need sessions (e.g. API) then use `array`
+# If you write a website, use `cookie` or store sessions in database.
+SESSION_DRIVER=cookie
```

Comment:
APIs don't use the session, but sessions for websites do not work with "array". At least now it works out of the box for both scenarios.